sample/build.gradle

```diff
@@ -39,14 +39,13 @@ android {
         versionName "1.0"
     }
 
-    compileOptions {
-        targetCompatibility JavaVersion.VERSION_1_8
-        sourceCompatibility JavaVersion.VERSION_1_8
-    }
-
     sourceSets {
         test.java.srcDirs += "src/test/kotlin"
     }
+    compileOptions {
+        targetCompatibility 1.8
+        sourceCompatibility 1.8
```

Comment:
What's wrong with using JavaVersion constants here?

Reply:
Android Studio 3.0 uses Java 8 as default language version.

Reply:
I'm aware of that, I was just wondering about the replacement of `JavaVersion.VERSION_1_8` with the String literal `1.8` here!

Reply:
Looking through AOSP, they mostly use the convention **VERSION_x_x**: http://androidxref.com/8.0.0_r4/s?n=25&start=50&sort=relevancy&q=targetCompatibility&project=art%2Cbionic%2Cbootable%2Cbuild%2Ccts%2Cdalvik%2Cdevelopers%2Cdevelopment%2Cdevice%2Cdocs%2Cexternal%2Cframeworks%2Chardware%2Ckernel%2Clibcore%2Clibnativehelper%2Cpackages%2Cpdk%2Cplatform_testing%2Cprebuilts%2Csdk%2Csystem%2Ctest%2Ctoolchain%2Ctools

I believe your way works just as well...